frontend/src/pages/dashboard/components/user-profile-info/user-profile-info.tsx

```diff
@@ -12,17 +17,23 @@ const UserProfileInfo: React.FC = () => {
   });
 
   return (
-    <div className={styles['container']}>
-      <div className={styles['header']}>Home</div>
-      <div className={styles['user-info']}>
-        <div className={styles['avatar-container']}>
-          <img src={avatar} alt="avatar" className={styles['avatar']} />
+    <Sidebar>
+      <SidebarHeader>
+        <div className={styles['info']}>
+          <span>Home</span>
         </div>
-        <span className={styles['user-name']}>
-          {(authenticatedUser as UserAuthResponseDto).fullName}
-        </span>
-      </div>
-    </div>
+      </SidebarHeader>
+      <SidebarBody>
+        <div className={styles['user-info']}>
+          <div className={styles['avatar-container']}>
+            <img src={avatar} alt="avatar" className={styles['avatar']} />
+          </div>
+          <span className={styles['user-name']}>
+            {(authenticatedUser as UserAuthResponseDto).fullName}
```

Comment:
let's move this cast to useAppSelector
```suggestion
            {authenticatedUser.fullName}
```

Reply:
done